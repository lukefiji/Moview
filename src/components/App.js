import React from 'react';

const App = () => {
  return (
    <div>
      <h1>Moview</h1>
      <p className="attribution__disclaimer">
        This product uses the TMDb API but is not endorsed or certified by TMDb.
      </p>
    </div>
  );
};

export default App;
