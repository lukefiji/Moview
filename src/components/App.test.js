import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  const app = shallow(<App />);

  it('should render properly', () => {
    expect(app).toMatchSnapshot();
  });

  it('should have a Movie DB disclaimer', () => {
    expect(app.find('.attribution__disclaimer').text()).toEqual(
      'This product uses the TMDb API but is not endorsed or certified by TMDb.'
    );
  });
});
