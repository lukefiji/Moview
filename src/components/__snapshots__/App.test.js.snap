// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Moview
</h1>,
        <p
          className="attribution__disclaimer"
>
          This product uses the TMDb API but is not endorsed or certified by TMDb.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Moview",
        },
        "ref": null,
        "rendered": "Moview",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "This product uses the TMDb API but is not endorsed or certified by TMDb.",
          "className": "attribution__disclaimer",
        },
        "ref": null,
        "rendered": "This product uses the TMDb API but is not endorsed or certified by TMDb.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Moview
</h1>,
          <p
            className="attribution__disclaimer"
>
            This product uses the TMDb API but is not endorsed or certified by TMDb.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Moview",
          },
          "ref": null,
          "rendered": "Moview",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "This product uses the TMDb API but is not endorsed or certified by TMDb.",
            "className": "attribution__disclaimer",
          },
          "ref": null,
          "rendered": "This product uses the TMDb API but is not endorsed or certified by TMDb.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
